proposals/dataset-record/README.md

```diff
@@ -0,0 +1,70 @@
+(todo: port this to be a proposal in Features API) that includes 'Dataset Fields', and shares most all the same fields.
+
+Dataset Records are represented in JSON format and are very flexible. Any JSON object that contains all the
+required fields is a valid Record.
+
+- Examples:
+  - See this [example](./examples/record-meetlocaties-example.json) that contains more fields and links.
+- JSON Schema: TODO
+
+
+## Dataset Record Fields
+
+The core Record fields for a 'Dataset Record' remain the same as in the core [OGC Record](ogc-record-geojson-spec.md), with the
+exact same Item fields as [specified there](ogc-record-geojson-spec.md#item-fields). (TODO: Link to main spec when Peter's refactor lands)
+
+### Datset Record Property Fields
+
+The property fields are where the Dataset Record has more requirements. It uses all the same core Record definitions, but adds in 
+more requirements and a couple defaults. 
+
+| Field Name        | Required in Core Record | Required in Dataset Record | Description                                                                                                                                                 |
+|-------------------|-------------------------|----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type              | M                       | M                          | Denotes the resource type of the record. For the dataset record this is **required** to be `dataset`.                                                         |
+| title             | M                       | M                          | A short descriptive human-readable one-line title for the Collection.                                                                                       |
+| description       | O                       | M                          | Detailed multi-line description to fully explain the Collection. |
+| keywords          | O                       | M                          | List of keywords describing the Collection.                                                                                                                 |
+| keywordsCodespace | O                       | O (defaults to XXX)        | A reference to a controlled vocabulary used for the keywords property.                                                                                      |
+| language          | O                       | O (defaults to english)    | The natural language used for textual values (i.e. titles, descriptions, etc) that the collection information is given in.                                  |
+| externalId        | O                       | O                          | Identifier for the Collection that is unique across the provider.                                                                                           |
```

Comment:
This is a comment regarding the draft Records API - but Is there a reason this can't be `id`? the prefix `external` is confusing unless it has some other meaning that's not stated. 

And I suggest it should be `M`